examples/nlp/text_normalization_as_tagging/prepare_dataset_en.sh

```diff
@@ -0,0 +1,255 @@
+do
+    python ${NEMO_PATH}/examples/nlp/text_normalization_as_tagging/utils/extract_giza_alignments.py \
+      --mode=itn \
+      --giza_dir=${subfolder} \
+      --out_filename=itn.out \
+      --giza_suffix=A3.final \
+      --lang=en &
+done
+wait
+
+for subfolder in ${ALIGNMENT_DIR}/*
+do
+    paste -d"\t" ${subfolder}/freq ${subfolder}/itn.out > ${subfolder}/itn.out2
+    awk 'BEGIN {FS="\t"} match($3, " "){print $0}' < ${subfolder}/itn.out2 | sort -rn > ${subfolder}/itn.debug
+done
+
+
+python ${NEMO_PATH}/examples/nlp/text_normalization_as_tagging/utils/prepare_corpora_after_alignment.py \
+  --mode=get_replacement_vocab \
+  --giza_dir=${ALIGNMENT_DIR} \
+  --alignment_filename=itn.out2 \
+  --data_dir="" \
+  --vocab_filename=${WORK_DIR}/replacement_vocab_full.txt \
+  --out_filename="" \
+  --lang=${CORPUS_LANG}
+
+grep -v "0__" replacement_vocab_full.txt.verbatim | head -n 108 > replacement_vocab_verbatim.txt
+grep -v "0__" replacement_vocab_full.txt.time | head -n 148 > replacement_vocab_time.txt
+grep -v "0__" replacement_vocab_full.txt.telephone | head -n 52 > replacement_vocab_telephone.txt
+head -n 0 replacement_vocab_full.txt.plain > replacement_vocab_plain.txt
+grep -v "0__" replacement_vocab_full.txt.ordinal | head -n 251 > replacement_vocab_ordinal.txt
+grep -v "0__" replacement_vocab_full.txt.money | grep -v "a__" | head -n 532 > replacement_vocab_money.txt
+grep -v "0__" replacement_vocab_full.txt.measure | head -n 488 > replacement_vocab_measure.txt
+head -n 257 replacement_vocab_full.txt.letters > replacement_vocab_letters.txt
+grep -v "0__" replacement_vocab_full.txt.fraction | head -n 169 > replacement_vocab_fraction.txt
+head -n 276 replacement_vocab_full.txt.electronic > replacement_vocab_electronic.txt
+head -n 73 replacement_vocab_full.txt.digit > replacement_vocab_digit.txt
+grep -v "0__" replacement_vocab_full.txt.decimal | head -n 149 > replacement_vocab_decimal.txt
+grep -v "0__" replacement_vocab_full.txt.date | grep -v "[0-9]-[0-9]" | grep -v "[0-9]\,[0-9]" | grep -v "[0-9]\.[0-9]" | grep -v "[0-9]\/[0-9]" | head -n 554 > replacement_vocab_date.txt
+grep -v "0__" replacement_vocab_full.txt.cardinal | head -n 402 > replacement_vocab_cardinal.txt
+head -n 137 replacement_vocab_full.txt.address > replacement_vocab_address.txt
+
+cat replacement_vocab_address.txt \
+  replacement_vocab_cardinal.txt \
+  replacement_vocab_date.txt \
+  replacement_vocab_decimal.txt \
+  replacement_vocab_digit.txt \
+  replacement_vocab_electronic.txt \
+  replacement_vocab_fraction.txt \
+  replacement_vocab_letters.txt \
+  replacement_vocab_measure.txt \
+  replacement_vocab_money.txt \
+  replacement_vocab_ordinal.txt \
+  replacement_vocab_plain.txt \
+  replacement_vocab_telephone.txt \
+  replacement_vocab_time.txt \
+  replacement_vocab_verbatim.txt > replacement_vocab.select.txt
+
+python ${NEMO_PATH}/examples/nlp/text_normalization_as_tagging/utils/prepare_corpora_after_alignment.py \
```

Comment:
this could be shortened and put in a loop + comments to some of the commands are missing

Reply:
done